packages/@lwc/compiler/src/__tests__/fixtures/expected-dynamic-component.js

```diff
@@ -0,0 +1,43 @@
+import { load } from '@custom/loader';
+import { registerTemplate, registerComponent, LightningElement, registerDecorators } from 'lwc';
+
+function tmpl($api, $cmp, $slotset, $ctx) {
+    const {
+        dc: api_dynamic_component,
+        f: api_flatten
+    } = $api;
+    return api_flatten([api_dynamic_component("x-foo", $cmp.customCtor, {
```

Comment:
@diervo here we should use `sc` instead of `f`, which is faster, and flatten is only needed if there is an iteration.

Reply:
will fix all that in a later PR once we solve all the dynamic compilation stuff all together.